lib/src/base-styles/_nx-page.scss

```diff
@@ -71,10 +71,14 @@
       margin-left: var(--nx-spacing-2x);
     }
   }
-  
+
   .nx-threat-counter-container {
     margin-top: 0;
-  
+    display: contents;
+    .nx-threat-counter {
+      margin: 0 var(--nx-spacing-4x) var(--nx-spacing-2x) 0;
+    }
+
```

Comment:
How about this for threat counters in the page title tags area to get them to wrap? 
![image](https://user-images.githubusercontent.com/6968527/123647063-ad6b2c80-d81f-11eb-9ef9-9a94e6d974e8.png)

Reply:
> How about this for threat counters in the page title tags area to get them to wrap?

That's getting there but there are more style adjustments needed.  The indivdual counters should have their bottom margin removed.  The margin between the last counter and the first tag is too large.  @dturner32 please clarify what it's supposed to be.  Additionally there are a few margin styles on the `.nx-threat-counter-container` here (a top margin and a left margin) which have no effect due to `display: contents` and which should be removed

Reply:
> That's getting there but there are more style adjustments needed. The indivdual counters should have their bottom margin removed. The margin between the last counter and the first tag is too large. @dturner32 please clarify what it's supposed to be. Additionally there are a few margin styles on the `.nx-threat-counter-container` here (a top margin and a left margin) which have no effect due to `display: contents` and which should be removed

Should be done in https://github.com/sonatype/sonatype-react-shared-components/pull/418/commits/358fd340bc90acad3ef36c05159efa3af04461e4 which
- Removed the bottom margin on the counters
- Removed the right margin on the last counter
- Decreased the left margin on the tag following the counter container to make it the same distance as between counters

![image](https://user-images.githubusercontent.com/6968527/123671352-13fb4500-d836-11eb-8dd6-f03df6f0ebfa.png)

Reply:
See the relevant mockup: 
![image](https://user-images.githubusercontent.com/3630091/123680701-0387a200-d817-11eb-8a63-82821e950a37.png)

I believe the intent was that there be 24px of space between each _kind_ of indicator.  So you took too much out of the gap between the threat indicators and the tags.

Reply:
> I believe the intent was that there be 24px of space between each _kind_ of indicator. So you took too much out of the gap between the threat indicators and the tags.

Added the missing space back in https://github.com/sonatype/sonatype-react-shared-components/pull/418/commits/0b23e70fe31be91d7e5e360a7c2b43043a24644b

Reply:
That's on one side, but judging from master there's a similar matter on the other side: 
![image](https://user-images.githubusercontent.com/3630091/123688051-97f60280-d81f-11eb-827d-1486ca436255.png)

Reply:
> That's on one side, but judging from master there's a similar matter on the other side:

I think this fixes it https://github.com/sonatype/sonatype-react-shared-components/pull/418/commits/6b2972f9e5040d72a30027ad8b8e7ac58add94f8